app/src/main/java/com/github/onlynotesswent/ui/overview/editnote/EditMarkdownScreen.kt

```diff
@@ -101,7 +101,17 @@ fun EditMarkdownScreen(
           markdownContent = downloadedFile
           state.setMarkdown(markdownContent?.readText() ?: "")
         },
-        onFileNotFound = {},
+        onFileNotFound = {
+          // If the file is not found, create a new file, that will be uploaded when saved
+          val file = File(context.cacheDir, "${selectedNote!!.id}.md")
+          if (!file.exists()) {
+            file.createNewFile()
+          }
```

Comment:
This could be personal preference but I think using `.apply()` might be cleaner:
```
val file = File(context.cacheDir, "${selectedNote!!.id}.md").apply {
    if (!exists()) createNewFile()
}
```